pkg/testcase/restartservice.go

```diff
@@ -0,0 +1,21 @@
+package testcase
+
+import (
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestRestartService() {
+	product, err := shared.GetProduct()
+	Expect(err).NotTo(HaveOccurred(), "failed to get product")
+	c := factory.ClusterConfig(GinkgoT())
+
+	_, err = shared.ManageService(product, "restart", "server", c.ServerIPs)
```

Comment:
1. change '_, err' to 'ip, err'
2. rephrase error meesage to: 
```fmt.Sprintf("failed to restart service for server/agent with ip: %s", ip)```

Reply:
ow got you now
tks